.storybook/config.tsx

```diff
@@ -43,7 +43,7 @@ addDecorator(story => (
 ));
 
 // automatically import all files ending in *.stories.js
-const req = require.context('../src', true, /.stories.js$/);
+const req = require.context('../src', true, /.stories.(js|ts|tsx|mdx)$/);
```

Comment:
why mdx?

Reply:
It's a typo, we no need `mdx` right now but I think is something you should invest regarding the component documentation. Storybook has improved his auto-generated documentation, take a look here: 
- https://docs.google.com/document/d/1un6YX7xDKEKl5-MVb-egnOYN8dynb5Hf7mq0hipk8JE/edit
- https://medium.com/storybookjs/storybook-docs-sneak-peak-5be78445094a